README.md

```diff
@@ -113,6 +113,39 @@ Alternatively, using distributor and importer objects:
 
 ## Testing
 
+### Dependencies
+
+If you're using a katello development install from katello-deploy to develop
+and run runcible tests locally, you'll need to install a couple
+packages (namely pulp-python-plugins and pulp-ostree-plugins).
```

Comment:
Minor thought, this could be re-worded to leave out the direct Katello references and more generally discuss how to setup a test environment for the OSTree bits. This would also be useful to add to -- https://github.com/Katello/katello-deploy/blob/master/shells/runcible.sh (the repo setup not the docs)